api/v1/model/src/commonMain/kotlin/OptionalValue.kt

```diff
@@ -97,6 +90,15 @@ sealed interface OptionalValue<out T> {
         }
 }
 
+/**
+ * Return the [value][Present.value] if this [OptionalValue] is [Present], otherwise throw an
+ * [IllegalArgumentException].
+ */
+val <T> OptionalValue<T>.valueOrThrow: T get() {
```

Comment:
Would the [`@Transient` annotation](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/basic-serialization.md#transient-properties) maybe work as well?

Reply:
The new JSON schema generator uses reflection by default:
https://github.com/SMILEY4/schema-kenerator/tree/develop/schema-kenerator-reflection

We could try switching to the kotlinx.serialization implementation, but I don't want to do it in this PR:
https://github.com/SMILEY4/schema-kenerator/tree/develop/schema-kenerator-serialization

Reply:
> The new JSON schema generator uses reflection by default:

Ouch 🙄